docs/docs/products/cosmwasm-vm-overview.md

```diff
@@ -0,0 +1,69 @@
+# Composable CosmWasm VM
+
+## Overview of Composable CosmWasm VM (ccw-vm)
+
+Composable is the first team building a CosmWasm virtual machine (VM) outside the Cosmos ecosystem that 
+developers can use to build with all the benefits of CosmWasm (CW) and the Cosmos Network.
+
+Confio's current implementation of CW is written half in Rust and half in Golang, 
+which means it is not compatible with Substrate and has never been used outside Cosmos. 
+Also, Confio's CW VM excludes library components but consists of a single implementation targeting the Cosmos SDK.
+
+Our team is creating a portable library version of CosmWasm that can be hosted on different implementations to 
+reduce the effort required to port CosmWasm to other ecosystems. 
+[Composable CosmWasm VM] (ccw-vm) has two host implementations which 
+allow developers to easily integrate CosmWasm into their ecosystems.
+This ensures that ccw-vm is highly portable and has consistent execution semantics regardless of its host. 
+As a result, developers can integrate ccw-vm as a pallet, within a frontend app, or inside a CLI tool.
+
+[Composable CosmWasm VM]: https://github.com/ComposableFi/cosmwasm-vm
+
+## Value proposition for CosmWasm projects deployed on Cosmos
+
+Building a CosmWasm virtual machine (VM) will allow developers to run smart contracts directly on [Picasso], 
+our chain on Kusama. 
+This provides novel opportunities, such as enabling existing Cosmos projects and CW developers to run smart contracts in 
+the Polkadot & Kusama (DotSama) ecosystems simultaneously.
+
+Accessible, interoperable smart contract creation is crucial for developing ecosystems, especially nascent ecosystems, 
+that will play a significant role in the future of DeFi.
+
+[Picasso]: https://picasso.xyz/
+
+## Overview of Components
+
+The ccw-vm can be divided into three components:
```

Comment:
I am not sure about the term `ccw-vm` here. Because cosmwasm virtual machine is `cosmwasm-vm` which is our actual virtual machine implementation. But the components explained here is the components of `CosmWasm on Substrate` which includes 3 different programs. @KaiserKarel wdyt?

Reply:
Edited to specify "CosmWasm on Picasso" with input from @JafarAz